main.go

```diff
@@ -66,32 +66,23 @@ func main() {
 		log.Fatal(err)
 	}
 
+	// maxHealthCheckDurationMinutes is a heuristic for how long the full health check
+	// loop should take. This is used to push out the expiry of active silences to make
+	// sure they don't expire while we're in that loop; so err on the high side.
+	// TODO: Use above vars to calculate this
+	maxHealthCheckDurationMinutes := 15
```

Comment:
Non blocker: Should we make this a `time.Duration` since it has a string method for printing.

Reply:
I'm working on the PR to make this value calculated. I'll see about adopting this suggestion there.

Reply:
[Later] Yeah, that works nicely.